server/graphql/v1/CollectiveInterface.js

```diff
@@ -720,6 +720,11 @@ export const CollectiveInterfaceType = new GraphQLInterfaceType({
             type: PaymentMethodOrderFieldType,
             description: 'Order entries based on given column. Set to null for no ordering.',
           },
+          includeOrgCollectivePM: {
```

Comment:
Not a great name. I would: org -> organization, PM -> PaymentMethod.

`includeOrganizationCollectivePaymentMethod`-> it's verbose but better for me.